data/World/Dodongos Cavern MQ.json

```diff
@@ -42,7 +42,61 @@
+            "Dodongos Cavern MQ Upper Lizalfos Pot 3": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Upper Lizalfos Pot 4": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Torch Puzzle Room Pot 1": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Torch Puzzle Room Pot 2": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Before Upper Lizalfos Pot 1": "
+                can_blast_or_smash or can_use(Sticks) or has_fire_source or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Before Upper Lizalfos Pot 2": "
+                can_blast_or_smash or can_use(Sticks) or has_fire_source or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ After Upper Lizalfos Pot 1": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ After Upper Lizalfos Pot 2": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Staircase Crate Bottom Left": "True",
+            "Dodongos Cavern MQ Staircase Crate Bottom Right": "True",
+            #whatever was used to destroy the boulder to activate the elevator can be used to access the top of the staircase
+            "Dodongos Cavern MQ Staircase Crate Mid Left": "True",
+            "Dodongos Cavern MQ Staircase Crate Mid Right": "True",
+            "Dodongos Cavern MQ Staircase Crate Top Left": "True",
+            "Dodongos Cavern MQ Staircase Crate Top Right": "True",
+            "Dodongos Cavern MQ Larvae Room Crate 1": "can_use(Sticks) or has_fire_source",
+            "Dodongos Cavern MQ Larvae Room Crate 2": "can_use(Sticks) or has_fire_source",
+            "Dodongos Cavern MQ Larvae Room Crate 3": "can_use(Sticks) or has_fire_source",
+            "Dodongos Cavern MQ Larvae Room Crate 4": "can_use(Sticks) or has_fire_source",
+            "Dodongos Cavern MQ Larvae Room Crate 5": "can_use(Sticks) or has_fire_source",
+            "Dodongos Cavern MQ Larvae Room Crate 6": "can_use(Sticks) or has_fire_source",
```

Comment:
We have a logic helper "has_fire_source_with_torch" which is equivalent to what you've got here and should probably be used instead. I know that's what was written in the file already, and feel free to update that too. When that logic was originally written predates the helper, and I guess it was never updated.